src/test/java/org/takes/facets/auth/codecs/CcStrictTest.java

```diff
@@ -39,6 +39,11 @@
     /**
      * CcStrict can block empty urn.
      * @throws IOException If some problem inside
+     * @todo #877:30min replace all org.junit.Test occurrences in the project
+     *  by JUnit 5 equivalent org.junit.jupiter.api.Test and replaces
+     *  all expected Exceptions by it's lamba style. Check
```

Comment:
@oridan Please add that we should use `Assertions.assertThrows` in assertions expecting exceptions and correct "lambda" typo